src/rustup-utils/src/lib.rs

```diff
@@ -1,13 +1,15 @@
 #![feature(core_intrinsics)] // For type_name().
 #![feature(fundamental)]
+#![feature(step_by)]
```

Comment:
Yeah, it just happened to be used by the sha code and didn't seem worth messing with, but it should be easy to remove. I'll look at it.